### ISSUE TITLE
Synonyms marked as diseases should never be EXACT

### ISSUE THREAD

**Issue by cmungall** [obophenotype/human-phenotype-ontology#2466]
```

[Term]
id: HP:0000108 ! Renal corticomedullary cysts
name: Renal corticomedullary cysts
synonym: "Corticomedullary renal cysts" EXACT []
***synonym: "Renal corticomedullary cystic disease" EXACT []

[Term]
id: HP:0000113 ! Polycystic kidney dysplasia
name: Polycystic kidney dysplasia
synonym: "Enlarged polycystic kidneys" EXACT []
***synonym: "Polycystic kidney disease" EXACT []
synonym: "Polycystic kidneys" EXACT []

[Term]
id: HP:0000124 ! Renal tubular dysfunction
name: Renal tubular dysfunction
synonym: "Renal tubular defect" EXACT []
***synonym: "Renal tubular disease" EXACT []

[Term]
id: HP:0000147 ! Polycystic ovaries
name: Polycystic ovaries
synonym: "Polycystic ovary" EXACT []
***synonym: "Polycystic ovary disease" EXACT []
synonym: "Sclerocystic ovaries" EXACT []

[Term]
id: HP:0000166 ! Severe periodontitis
name: Severe periodontitis
synonym: "Severe gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]
***synonym: "Severe periodontal disease" EXACT [orcid.org/0000-0001-5889-4463]
synonym: "Severe pyorrhea" EXACT [orcid.org/0000-0001-5889-4463]

[Term]
id: HP:0000488 ! Retinopathy
name: Retinopathy
***synonym: "Noninflammatory retina disease" EXACT layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0000820 ! Abnormality of the thyroid gland
name: Abnormality of the thyroid gland
synonym: "Abnormality of the thyroid gland" EXACT layperson []
synonym: "Thyroid abnormality" EXACT layperson []
***synonym: "Thyroid disease" EXACT layperson []

[Term]
id: HP:0001271 ! Polyneuropathy
name: Polyneuropathy
***synonym: "Peripheral nerve disease" EXACT layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0001291 ! Abnormality of the cranial nerves
name: Abnormality of the cranial nerves
synonym: "Cranial nerve abnormality" RELATED []
***synonym: "Cranial nerve disease" EXACT []
synonym: "Cranial nerve involvement" EXACT []

[Term]
id: HP:0001677 ! Coronary artery disease
name: Coronary artery disease
***synonym: "Coronary disease" EXACT []

[Term]
id: HP:0001907 ! Thromboembolism
name: Thromboembolism
synonym: "Embolism and thrombosis" RELATED []
***synonym: "Thromboembolic disease" EXACT []
synonym: "Thromboembolic events" EXACT []

[Term]
id: HP:0002020 ! Gastroesophageal reflux
name: Gastroesophageal reflux
synonym: "Acid reflux" EXACT layperson []
***synonym: "Acid reflux disease" EXACT layperson []
synonym: "GERD" 0045077 [HPO:skoehler]
***synonym: "Gastroesophageal reflux disease" EXACT []
synonym: "Heartburn" EXACT layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0002091 ! Restrictive ventilatory defect
name: Restrictive ventilatory defect
***synonym: "Restrictive lung disease" EXACT [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0002111 ! Restrictive deficit on pulmonary function testing
name: Restrictive deficit on pulmonary function testing
synonym: "Restrictive deficit on pulmonary function tests" EXACT []
***synonym: "Restrictive respiratory disease" EXACT []
synonym: "Restrictive respiratory insufficiency'" EXACT []
synonym: "Restrictive respiratory syndrome" EXACT []

[Term]
id: HP:0002134 ! Abnormality of the basal ganglia
name: Abnormality of the basal ganglia
synonym: "Anomaly of the basal ganglia" EXACT []
***synonym: "Basal ganglia disease" EXACT []

[Term]
id: HP:0002251 ! Aganglionic megacolon
name: Aganglionic megacolon
synonym: "Congenital megacolon" EXACT []
***synonym: "Hirschsprung disease" EXACT []
synonym: "Hirschsprung megacolon" EXACT []
synonym: "Megacolon" EXACT []

[Term]
id: HP:0002366 ! Abnormal lower motor neuron morphology
name: Abnormal lower motor neuron morphology
***synonym: "Lower motor neuron disease" EXACT []
synonym: "Lower motor neuron manifestations" EXACT []
synonym: "Lower motor neuron signs" EXACT []

[Term]
id: HP:0002511 ! Alzheimer disease
name: Alzheimer disease
***synonym: "Alzheimer disease" EXACT layperson []
***synonym: "Late-onset form of familial Alzheimer disease" EXACT layperson []

[Term]
id: HP:0002608 ! Celiac disease
name: Celiac disease
***synonym: "Celiac disease" EXACT layperson []
synonym: "Celiac sprue" EXACT []

[Term]
id: HP:0002621 ! Atherosclerosis
name: Atherosclerosis
***synonym: "Atherosclerotic cardiovascular disease" EXACT [http://www.mayoclinic.org/diseases-conditions/arteriosclerosis-atherosclerosis/basics/definition/con-20026972?utm_source=google&amp;utm_medium=abstract&amp;utm_content=atherosclerosis&amp;utm_campaign=knowledge-panel, orcid.org/0000-0001-5208-3432]
synonym: "Plaque build-up in arteries" BROAD layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0002758 ! Osteoarthritis
name: Osteoarthritis
***synonym: "Degenerative joint disease" EXACT layperson []

[Term]
id: HP:0002960 ! Autoimmunity
name: Autoimmunity
synonym: "Autoimmune condition" EXACT []
***synonym: "Autoimmune disease" EXACT layperson []
synonym: "Autoimmune disorders" EXACT layperson []
synonym: "Autoimmunity" EXACT layperson []

[Term]
id: HP:0003198 ! Myopathy
name: Myopathy
***synonym: "Muscle tissue disease" EXACT layperson [orcid.org/0000-0001-5208-3432]
synonym: "Myopathic changes" EXACT []

[Term]
id: HP:0003774 ! Stage 5 chronic kidney disease
name: Stage 5 chronic kidney disease
synonym: "Chronic renal failure" EXACT []
***synonym: "End stage renal disease" EXACT []
synonym: "End stage renal failure" EXACT []
***synonym: "End-stage renal disease" EXACT []
synonym: "End-stage renal failure" EXACT [HPO:skoehler]
synonym: "Renal failure, endstage" RELATED layperson [HPO:skoehler]
***synonym: "Stage 5 chronic kidney disease" EXACT layperson []

[Term]
id: HP:0004950 ! Peripheral arterial disease
name: Peripheral arterial disease
***synonym: "Peripheral artery disease" EXACT [http://orcid.org/0000-0001-6908-9849]
synonym: "Peripheral vascular disease" RELATED [HPO:probinson]

[Term]
id: HP:0005181 ! Premature coronary artery disease
name: Premature coronary artery disease
***synonym: "Premature coronary artery disease" EXACT layperson []

[Term]
id: HP:0005315 ! Peripheral artery occlusive disease
name: Peripheral artery occlusive disease
***synonym: "Occlusive arterial disease" EXACT []
synonym: "Occlusive vascular disease" RELATED []
synonym: "Peripheral vascular insufficiency" RELATED []

[Term]
id: HP:0005743 ! Avascular necrosis of the capital femoral epiphysis
name: Avascular necrosis of the capital femoral epiphysis
synonym: "Coxa plana" RELATED []
synonym: "Legg-Calve-Perthes syndrome" EXACT []
***synonym: "Legg-Perthes disease" EXACT []
synonym: "Morbus Legg-Calve-Perthes" EXACT []
synonym: "Osteochondrosis of the femoral head" EXACT []
synonym: "Perthes-like femoral head changes" EXACT []

[Term]
id: HP:0006528 ! Chronic lung disease
name: Chronic lung disease
***synonym: "Chronic lung disease" EXACT layperson []

[Term]
id: HP:0006530 ! Interstitial pulmonary abnormality
name: Interstitial pulmonary abnormality
***synonym: "Interstitial lung disease" EXACT []
***synonym: "Interstitial pulmonary disease" EXACT []

[Term]
id: HP:0006536 ! Obstructive lung disease
name: Obstructive lung disease
***synonym: "Obstructive lung disease" EXACT layperson []

[Term]
id: HP:0006775 ! Multiple myeloma
name: Multiple myeloma
***synonym: "Kahler's disease" EXACT [http://orcid.org/0000-0001-6908-9849]
synonym: "Plasmocytoma" RELATED []

[Term]
id: HP:0006802 ! Abnormal anterior horn cell morphology
name: Abnormal anterior horn cell morphology
synonym: "Abnormality of the anterior horn cell" BROAD []
synonym: "Abnormality of the anterior horn cells" EXACT []
synonym: "Anomaly of the anterior horn cells" EXACT []
***synonym: "Anterior horn cell disease" EXACT []

[Term]
id: HP:0007354 ! Amyotrophic lateral sclerosis
name: Amyotrophic lateral sclerosis
***synonym: "Lou gehrig's disease" EXACT layperson [http://orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0007570 ! Hyperkeratosis lenticularis perstans
name: Hyperkeratosis lenticularis perstans
***synonym: "Flegel disease" EXACT []

[Term]
id: HP:0008207 ! Primary adrenal insufficiency
name: Primary adrenal insufficiency
***synonym: "Addison disease" EXACT []
***synonym: "Addison's disease" EXACT []
synonym: "Adrenocortical insufficiency" EXACT []
synonym: "Primary adrenocortical failure" EXACT []

[Term]
id: HP:0008419 ! Intervertebral disc degeneration
name: Intervertebral disc degeneration
synonym: "Degeneration of intervertebral disks" EXACT []
***synonym: "Degenerative disc disease" EXACT [http://orcid.org/0000-0001-6908-9849]
synonym: "Degenerative intervertebral disk" EXACT []

[Term]
id: HP:0010889 ! Morbus Kienboeck
name: Morbus Kienboeck
***synonym: "Kienbock's disease" EXACT []
synonym: "Lunatomalacia" EXACT []

[Term]
id: HP:0010890 ! Morbus Osgood-Schlatter
name: Morbus Osgood-Schlatter
***synonym: "Osgood Schlatter disease" EXACT []

[Term]
id: HP:0010891 ! Morbus Scheuermann
name: Morbus Scheuermann
***synonym: "Calve disease" EXACT []
synonym: "Juvenile Osteochondrosis of the spine" EXACT []
***synonym: "Scheuermann disease" EXACT []
synonym: "Scheuermann kyphosis" EXACT []
synonym: "Sherman's Disease" EXACT []

[Term]
id: HP:0010959 ! Congenital cystic adenomatoid malformation of the lung
name: Congenital cystic adenomatoid malformation of the lung
synonym: "CCAM" 0045077 []
***synonym: "Congenital cystic disease of the lung" EXACT []
***synonym: "Cystic adenomatoid lung disease" EXACT []

[Term]
id: HP:0011034 ! Amyloidosis
name: Amyloidosis
***synonym: "Amyloid disease" EXACT [orcid.org/0000-0001-6908-9849]

[Term]
id: HP:0011058 ! Generalized periodontitis
name: Generalized periodontitis
synonym: "Generalized gum disease" BROAD [orcid.org/0000-0001-5889-4463]
***synonym: "Generalized periodontal disease" EXACT [orcid.org/0000-0001-5889-4463]
synonym: "Widespread gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]

[Term]
id: HP:0011059 ! Localized periodontitis
name: Localized periodontitis
synonym: "Limited area of gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]
synonym: "Localized gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]
***synonym: "Localized periodontal disease" EXACT [orcid.org/0000-0001-5889-4463]

[Term]
id: HP:0012189 ! Hodgkin lymphoma
name: Hodgkin lymphoma
***synonym: "Hodgkin disease" EXACT []
synonym: "Hodgkin's lymphoma" EXACT []

[Term]
id: HP:0012436 ! Nonocclusive coronary artery disease
name: Nonocclusive coronary artery disease
***synonym: "Non-occlusive coronary artery disease" EXACT []
synonym: "Non-occlusive coronary artery stenosis" RELATED []
synonym: "Nonocclusive coronary artery stenosis" RELATED []

[Term]
id: HP:0012579 ! Minimal change glomerulonephritis
name: Minimal change glomerulonephritis
***synonym: "Minimal change disease" EXACT []
synonym: "Minimal change nephropathy" EXACT []

[Term]
id: HP:0012622 ! Chronic kidney disease
name: Chronic kidney disease
***synonym: "Chronic kidney disease" EXACT layperson []
synonym: "Loss of renal function" RELATED []
synonym: "Progressive renal failure" RELATED []
synonym: "Progressive renal insufficiency" RELATED []
synonym: "Renal failure, progressive" RELATED [HPO:skoehler]
synonym: "Renal insufficiency, progressive" RELATED []

[Term]
id: HP:0012626 ! Stage 4 chronic kidney disease
name: Stage 4 chronic kidney disease
***synonym: "Stage 4 chronic kidney disease" EXACT layperson []
```

I suppose that some of these are marked as diseases in actual rdfs:label, reflecting a deeper problem here. I know some diseases can be features but seems we need a better strategy here

**Comment by nicolevasilevsky** [comment]
should they be related? I can fix it.

But what about id: HP:0012622 ! Chronic kidney disease? It has an exact synonym that is the same as the label because that is how we are now assigning the layperson tags, we just duplicate the label as an exact syn and add the layperson tag

**Comment by cmungall** [comment]
To try and separate these two issues, here is a smaller list, where the label does not have the string disease

```

[Term]
id: HP:0000108 ! Renal corticomedullary cysts
name: Renal corticomedullary cysts
synonym: "Corticomedullary renal cysts" EXACT []
***synonym: "Renal corticomedullary cystic disease" EXACT []

[Term]
id: HP:0000113 ! Polycystic kidney dysplasia
name: Polycystic kidney dysplasia
synonym: "Enlarged polycystic kidneys" EXACT []
***synonym: "Polycystic kidney disease" EXACT []
synonym: "Polycystic kidneys" EXACT []

[Term]
id: HP:0000124 ! Renal tubular dysfunction
name: Renal tubular dysfunction
synonym: "Renal tubular defect" EXACT []
***synonym: "Renal tubular disease" EXACT []

[Term]
id: HP:0000147 ! Polycystic ovaries
name: Polycystic ovaries
synonym: "Polycystic ovary" EXACT []
***synonym: "Polycystic ovary disease" EXACT []
synonym: "Sclerocystic ovaries" EXACT []

[Term]
id: HP:0000166 ! Severe periodontitis
name: Severe periodontitis
synonym: "Severe gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]
***synonym: "Severe periodontal disease" EXACT [orcid.org/0000-0001-5889-4463]
synonym: "Severe pyorrhea" EXACT [orcid.org/0000-0001-5889-4463]

[Term]
id: HP:0000488 ! Retinopathy
name: Retinopathy
***synonym: "Noninflammatory retina disease" EXACT layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0000820 ! Abnormality of the thyroid gland
name: Abnormality of the thyroid gland
synonym: "Abnormality of the thyroid gland" EXACT layperson []
synonym: "Thyroid abnormality" EXACT layperson []
***synonym: "Thyroid disease" EXACT layperson []

[Term]
id: HP:0001271 ! Polyneuropathy
name: Polyneuropathy
***synonym: "Peripheral nerve disease" EXACT layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0001291 ! Abnormality of the cranial nerves
name: Abnormality of the cranial nerves
synonym: "Cranial nerve abnormality" RELATED []
***synonym: "Cranial nerve disease" EXACT []
synonym: "Cranial nerve involvement" EXACT []

[Term]
id: HP:0001907 ! Thromboembolism
name: Thromboembolism
synonym: "Embolism and thrombosis" RELATED []
***synonym: "Thromboembolic disease" EXACT []
synonym: "Thromboembolic events" EXACT []

[Term]
id: HP:0002020 ! Gastroesophageal reflux
name: Gastroesophageal reflux
synonym: "Acid reflux" EXACT layperson []
***synonym: "Acid reflux disease" EXACT layperson []
synonym: "GERD" 0045077 [HPO:skoehler]
***synonym: "Gastroesophageal reflux disease" EXACT []
synonym: "Heartburn" EXACT layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0002091 ! Restrictive ventilatory defect
name: Restrictive ventilatory defect
***synonym: "Restrictive lung disease" EXACT [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0002111 ! Restrictive deficit on pulmonary function testing
name: Restrictive deficit on pulmonary function testing
synonym: "Restrictive deficit on pulmonary function tests" EXACT []
***synonym: "Restrictive respiratory disease" EXACT []
synonym: "Restrictive respiratory insufficiency'" EXACT []
synonym: "Restrictive respiratory syndrome" EXACT []

[Term]
id: HP:0002134 ! Abnormality of the basal ganglia
name: Abnormality of the basal ganglia
synonym: "Anomaly of the basal ganglia" EXACT []
***synonym: "Basal ganglia disease" EXACT []

[Term]
id: HP:0002251 ! Aganglionic megacolon
name: Aganglionic megacolon
synonym: "Congenital megacolon" EXACT []
***synonym: "Hirschsprung disease" EXACT []
synonym: "Hirschsprung megacolon" EXACT []
synonym: "Megacolon" EXACT []

[Term]
id: HP:0002366 ! Abnormal lower motor neuron morphology
name: Abnormal lower motor neuron morphology
***synonym: "Lower motor neuron disease" EXACT []
synonym: "Lower motor neuron manifestations" EXACT []
synonym: "Lower motor neuron signs" EXACT []

[Term]
id: HP:0002621 ! Atherosclerosis
name: Atherosclerosis
***synonym: "Atherosclerotic cardiovascular disease" EXACT [http://www.mayoclinic.org/diseases-conditions/arteriosclerosis-atherosclerosis/basics/definition/con-20026972?utm_source=google&amp;utm_medium=abstract&amp;utm_content=atherosclerosis&amp;utm_campaign=knowledge-panel, orcid.org/0000-0001-5208-3432]
synonym: "Plaque build-up in arteries" BROAD layperson [orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0002758 ! Osteoarthritis
name: Osteoarthritis
***synonym: "Degenerative joint disease" EXACT layperson []

[Term]
id: HP:0002960 ! Autoimmunity
name: Autoimmunity
synonym: "Autoimmune condition" EXACT []
***synonym: "Autoimmune disease" EXACT layperson []
synonym: "Autoimmune disorders" EXACT layperson []
synonym: "Autoimmunity" EXACT layperson []

[Term]
id: HP:0003198 ! Myopathy
name: Myopathy
***synonym: "Muscle tissue disease" EXACT layperson [orcid.org/0000-0001-5208-3432]
synonym: "Myopathic changes" EXACT []

[Term]
id: HP:0005743 ! Avascular necrosis of the capital femoral epiphysis
name: Avascular necrosis of the capital femoral epiphysis
synonym: "Coxa plana" RELATED []
synonym: "Legg-Calve-Perthes syndrome" EXACT []
***synonym: "Legg-Perthes disease" EXACT []
synonym: "Morbus Legg-Calve-Perthes" EXACT []
synonym: "Osteochondrosis of the femoral head" EXACT []
synonym: "Perthes-like femoral head changes" EXACT []

[Term]
id: HP:0006530 ! Interstitial pulmonary abnormality
name: Interstitial pulmonary abnormality
***synonym: "Interstitial lung disease" EXACT []
***synonym: "Interstitial pulmonary disease" EXACT []

[Term]
id: HP:0006775 ! Multiple myeloma
name: Multiple myeloma
***synonym: "Kahler's disease" EXACT [http://orcid.org/0000-0001-6908-9849]
synonym: "Plasmocytoma" RELATED []

[Term]
id: HP:0006802 ! Abnormal anterior horn cell morphology
name: Abnormal anterior horn cell morphology
synonym: "Abnormality of the anterior horn cell" BROAD []
synonym: "Abnormality of the anterior horn cells" EXACT []
synonym: "Anomaly of the anterior horn cells" EXACT []
***synonym: "Anterior horn cell disease" EXACT []

[Term]
id: HP:0007354 ! Amyotrophic lateral sclerosis
name: Amyotrophic lateral sclerosis
***synonym: "Lou gehrig's disease" EXACT layperson [http://orcid.org/0000-0001-5208-3432]

[Term]
id: HP:0007570 ! Hyperkeratosis lenticularis perstans
name: Hyperkeratosis lenticularis perstans
***synonym: "Flegel disease" EXACT []

[Term]
id: HP:0008207 ! Primary adrenal insufficiency
name: Primary adrenal insufficiency
***synonym: "Addison disease" EXACT []
***synonym: "Addison's disease" EXACT []
synonym: "Adrenocortical insufficiency" EXACT []
synonym: "Primary adrenocortical failure" EXACT []

[Term]
id: HP:0008419 ! Intervertebral disc degeneration
name: Intervertebral disc degeneration
synonym: "Degeneration of intervertebral disks" EXACT []
***synonym: "Degenerative disc disease" EXACT [http://orcid.org/0000-0001-6908-9849]
synonym: "Degenerative intervertebral disk" EXACT []

[Term]
id: HP:0010889 ! Morbus Kienboeck
name: Morbus Kienboeck
***synonym: "Kienbock's disease" EXACT []
synonym: "Lunatomalacia" EXACT []

[Term]
id: HP:0010890 ! Morbus Osgood-Schlatter
name: Morbus Osgood-Schlatter
***synonym: "Osgood Schlatter disease" EXACT []

[Term]
id: HP:0010891 ! Morbus Scheuermann
name: Morbus Scheuermann
***synonym: "Calve disease" EXACT []
synonym: "Juvenile Osteochondrosis of the spine" EXACT []
***synonym: "Scheuermann disease" EXACT []
synonym: "Scheuermann kyphosis" EXACT []
synonym: "Sherman's Disease" EXACT []

[Term]
id: HP:0010959 ! Congenital cystic adenomatoid malformation of the lung
name: Congenital cystic adenomatoid malformation of the lung
synonym: "CCAM" 0045077 []
***synonym: "Congenital cystic disease of the lung" EXACT []
***synonym: "Cystic adenomatoid lung disease" EXACT []

[Term]
id: HP:0011034 ! Amyloidosis
name: Amyloidosis
***synonym: "Amyloid disease" EXACT [orcid.org/0000-0001-6908-9849]

[Term]
id: HP:0011058 ! Generalized periodontitis
name: Generalized periodontitis
synonym: "Generalized gum disease" BROAD [orcid.org/0000-0001-5889-4463]
***synonym: "Generalized periodontal disease" EXACT [orcid.org/0000-0001-5889-4463]
synonym: "Widespread gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]

[Term]
id: HP:0011059 ! Localized periodontitis
name: Localized periodontitis
synonym: "Limited area of gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]
synonym: "Localized gum disease" BROAD layperson [orcid.org/0000-0001-5889-4463]
***synonym: "Localized periodontal disease" EXACT [orcid.org/0000-0001-5889-4463]

[Term]
id: HP:0012189 ! Hodgkin lymphoma
name: Hodgkin lymphoma
***synonym: "Hodgkin disease" EXACT []
synonym: "Hodgkin's lymphoma" EXACT []

[Term]
id: HP:0012579 ! Minimal change glomerulonephritis
name: Minimal change glomerulonephritis
***synonym: "Minimal change disease" EXACT []
synonym: "Minimal change nephropathy" EXACT []
```

**Comment by nicolevasilevsky** [comment]
@cmungall should these be broad synonyms?

If yes, I can fix

**Comment by pnrobinson** [comment]
@cmungall @nicolevasilevsky I think that these are RELATED SYNONYMs because they are referring to diseases rather than phenotypes. However,the list above is also a mixed bag. Maybe we should go through this by phone?

**Comment by cmungall** [comment]
Happy to help further but I don't think I'm critical for moving this along...

**Comment by nicolevasilevsky** [comment]
@pnrobinson - we have a meeting scheduled soon, let's talk about this. I'll create a github project for our call and add this ticket.

**Comment by cmungall** [comment]
For reference, this is the list of terms ending with disease$ in the primary label. In the past we have discussed these and kept them in HPO as they can be features of other diseases (but if this is the criteria, do we not end up replicating half of mondo...?)

```
 HP:0001580 ! Pigmented micronodular adrenocortical disease
 HP:0001677 ! Coronary artery disease
 HP:0002511 ! Alzheimer disease
 HP:0002608 ! Celiac disease
 HP:0002611 ! Cholestatic liver disease
 HP:0003774 ! Stage 5 chronic kidney disease
 HP:0005181 ! Premature coronary artery disease
 HP:0005237 ! Degenerative liver disease
 HP:0005381 ! Recurrent meningococcal disease
 HP:0005524 ! Macrocytic hemolytic disease
 HP:0005948 ! Cystic lung disease
 HP:0006510 ! Chronic obstructive pulmonary disease
 HP:0006528 ! Chronic lung disease
 HP:0006536 ! Obstructive lung disease
 HP:0006552 ! Fibrocystic lung disease
 HP:0006557 ! Polycystic liver disease
 HP:0006566 ! Neonatal cholestatic liver disease
 HP:0006694 ! Early progressive calcific cardiac valvular disease
 HP:0006706 ! Cystic liver disease
 HP:0011437 ! Maternal autoimmune disease
 HP:0012436 ! Nonocclusive coronary artery disease
 HP:0012622 ! Chronic kidney disease
 HP:0012623 ! Stage 1 chronic kidney disease
 HP:0012624 ! Stage 2 chronic kidney disease
 HP:0012625 ! Stage 3 chronic kidney disease
 HP:0012626 ! Stage 4 chronic kidney disease
 HP:0030441 ! Anal margin Paget's disease
 HP:0100280 ! Crohn's disease
 HP:0100647 ! Graves disease
```

**Comment by pnrobinson** [comment]
These items are on a spectrum. Stage n chronic kidney disease can be easily conceptualized as a phenotypic feature of disease, while Grave's disease requires supplemental imagination stretching. Once MONDO is mature, we can try and extend our annotation model to use MONDO terms instead of HPO terms in our annotations for things such as Grave's disease. This will be of benefit also for rare disease algorithms. 
I would say though that we should not be worried too much by the use of a certain word or not ("disease"), and we should never say never say never. 
I do not therefore think that we can automate the revision of these items. I would suggest that we take no general action now but be aware of this and try to make quick progress on Mondo so we can do this in a more elegant way in the future.

**Comment by pnrobinson** [comment]
I have gone through the entire list. In many cases, "xxx....disease" is truly used as a way of describing what we call a phenotype. In cases where a disease name was used as an EXACT syn of a phenotype, I changed to RELATED synonym. There are several items that we will need to revisit after MONDO is more mature so that we reference diseases rather than phenotypes (e.g., Crohn disease), but this is not realistic until we have made progress on the common disease formats etc.
